tests/helpers/load_state_tests.py

```diff
@@ -24,6 +27,7 @@ class NoTestsFound(Exception):
 
 
 def run_blockchain_st_test(test_case: Dict, load: Load) -> None:
+    load.fork.fork_criteria = ByBlockNumber(0)
```

Comment:
This line feels like a hack, but I'm not sure what the correct solution would look like. Open to suggestions!